.resinci.yml

```diff
@@ -0,0 +1,17 @@
+reviewers: 1
+docker:
+  builds:
+    - path: .
+      dockerfile: Dockerfile
+      docker_repo: balena/pinejs-sut
+      publish: false
+npm:
+  run: true
+  platforms:
+    - name: linux
+      os: alpine
+      architecture: x86_64
+      node_versions:
+        - "12"
+        - "14"
+        - "16"
```

Comment:
I assume it's necessary to manually specify all of these now because it's a `docker` rather than `node` project officially? If so can you add `18` as well please as it's the next LTS and it'd be good to have it in the mix

Reply:
Added 18

Reply:
Adding 18 doesn't add it to the resin ci steps. Checking now the resin ci scripts.